thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/v2/AnomaliesResource.java

```diff
@@ -658,7 +655,7 @@ private AnomalyDetails getAnomalyDetails(MergedAnomalyResultDTO mergedAnomaly, D
 
     long anomalyStartTime = mergedAnomaly.getStartTime();
     long anomalyEndTime = mergedAnomaly.getEndTime();
-    TimeRange range = getTimeseriesOffsetedTimes(anomalyStartTime, anomalyEndTime, datasetConfig);
+    TimeRange range = getTimeseriesTraingingOffsetedTimes(anomalyStartTime, anomalyEndTime, datasetConfig);
```

Comment:
As per our discussion, you've introduced 0 offset in training because your function looks back 90 days. BUt this will work only for that specific function, and not for other DAILY functions. We can't make an assumption on automatic fetching of data at this place, and this place should be agnostic of logic specific to functions.
I would suggest, either keep it like it was before and just make offset 30 days (we will end up fetching 30+90 days), or introduce a new function in the backend for this kind of timeseries fetch.